eng/Signing.props

```diff
@@ -16,4 +16,12 @@
       -->
     <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.zip" />
   </ItemGroup>
+  <ItemGroup Condition="'$(SignAllBinaries)' == 'true'">
+    <!--
+      Sign all binaries in the artifacts directory.
+      -->
+    <ItemsToSign Remove="@(ItemsToSign)" />
+    <ItemsToSign Include="$(ArtifactsDir)bin\**\*" />
+    <ItemsToSign Include="$(ArtifactsDir)pub\**\*" />
```

Comment:
Are there duplications between pub and bin? Do we need to sign both places?

Reply:
`bin` contains the profiler assemblies only in the current usage; `pub` contains the results of `dotnet publish` for dotnet-monitor for each platform. There isn't much duplication except for the dependencies; however the signing task seems to be smart about it when submitting for signing by hashing the files to determine uniqueness, only submitting the unique set, and copying over duplicates with the same signed files.